### DISCUSSION TITLE
Is there a way to customize sidebar svg icons colors?

### DISCUSSION THREAD

**Discussion by ghost** [storybookjs/storybook#18336] [Help]
Hi, 
I was looking into customizing the look of storybook for my project. I see the them colorPrimary gets applied to one of the icons. Is there a way to customize the color for the rest of icons used on for the stories? I guess the generated class for the svgs will change with every storybook update, so I was wondering if there is a more direct way to target them via CSS within the manager-head.html file. Thank you

![image](https://user-images.githubusercontent.com/1149371/170525585-ea5dad2a-2e5a-453d-b59b-bf486061ea9f.png)

**Comment by vekunz** [comment]
I changed the color by settings this style in the `.storybook/manager-head.html` file:
```html
<style>
    #storybook-explorer-tree .sidebar-item svg {
        color: #008000;
    }
</style>
```

**Comment by ironmikesh** [comment]
A way to target a SINGLE svg in your Storybook side nav is to identify the id of the anchor link holding the svg you want to change.  (Right click, inspect, etc...) Then use that id to target the svg in the `.storybook/manager-head.html` stylings.  You may wish to handle both normal/enabled state as well as selected state.  Here's an example for say a button component that lives in a buttons folder - it might look something like this:
```
#components-buttons-button svg {
  color: red;
}
#components-buttons-button[data-selected='true'] svg {
  color: white;
}
```
Notes:
- the `[data-selected=true] one handles the selected state, which may be too dark for reader accessibility (depending on how you have set up the selected state of storybook nav items)
- the svg colors do not update on the fly. You will have to rebuild storybook to see the color changes.
